services/auth/source/ldap/source_sync.go

```diff
@@ -41,9 +39,15 @@ func (source *Source) Sync(ctx context.Context, updateExisting bool) error {
 	default:
 	}
 
-	sort.Slice(users, func(i, j int) bool {
-		return users[i].LowerName < users[j].LowerName
-	})
+	usernameUsers := make(map[string]*user_model.User, len(users))
+	mailUsers := make(map[string]*user_model.User, len(users))
+	existingUsers := make(map[int64]struct{})
+
+	for i := range users {
+		u := users[i]
```

Comment:
```suggestion
	for _, u := range users {
```

I guess Golang likes this syntax (and below) (well, not a must)

Reply:
fixed in e50f6d0c3880937ae5ff4745872402740e73724c and 99ee376ec49f92e239c4aed491443ba2e613c6ab